packages/l/libwebp/xmake.lua

```diff
@@ -7,38 +7,43 @@ package("libwebp")
+    add_versions("v1.2.0", "d37571723f531e5002667632c94fa18857ed7eb5b0561e3b49e913c3e0b1403e")
+    add_versions("v1.2.1", "7926985218c9e546069c2013dd93774aac3f012fd275247f82b0c119ec9a3801")
     add_versions("v1.2.2", "51e9297aadb7d9eb99129fe0050f53a11fcce38a0848fb2b0389e385ad93695e")
+    add_versions("v1.3.0", "dc9860d3fe06013266c237959e1416b71c63b36f343aae1d65ea9c94832630e1")
 
-    add_configs("anim_utils", {description = "Build animation utilities.", default = false, type = "boolean"})
-    add_configs("cwebp",      {description = "Build the cwebp command line tool.", default = false, type = "boolean"})
-    add_configs("dwebp",      {description = "Build the dwebp command line tool.", default = false, type = "boolean"})
-    add_configs("gif2webp",   {description = "Build the gif2webp conversion tool.", default = false, type = "boolean"})
-    add_configs("img2webp",   {description = "Build the img2webp animation tool.", default = false, type = "boolean"})
-    add_configs("vwebp",      {description = "Build the vwebp viewer tool.", default = false, type = "boolean"})
-    add_configs("webpinfo",   {description = "Build the webpinfo command line tool.", default = false, type = "boolean"})
-    add_configs("libwebpmux", {description = "Build the libwebpmux library.", default = false, type = "boolean"})
-    add_configs("webpmux",    {description = "Build the webpmux command line tool.", default = false, type = "boolean"})
-    add_configs("sharpyuv",   {description = "Build the sharpyuv library.", default = false, type = "boolean"})
-    add_configs("extras",     {description = "Build extras.", default = false, type = "boolean"})
+    add_configs("anim_utils", {description = "Build animation utilities.", default = true, type = "boolean"})
+    add_configs("cwebp",      {description = "Build the cwebp command line tool.", default = true, type = "boolean"})
+    add_configs("dwebp",      {description = "Build the dwebp command line tool.", default = true, type = "boolean"})
+    add_configs("gif2webp",   {description = "Build the gif2webp conversion tool.", default = true, type = "boolean"})
+    add_configs("img2webp",   {description = "Build the img2webp animation tool.", default = true, type = "boolean"})
+    add_configs("vwebp",      {description = "Build the vwebp viewer tool.", default = true, type = "boolean"})
+    add_configs("webpinfo",   {description = "Build the webpinfo command line tool.", default = true, type = "boolean"})
+    add_configs("libwebpmux", {description = "Build the libwebpmux library.", default = true, type = "boolean"})
+    add_configs("webpmux",    {description = "Build the webpmux command line tool.", default = true, type = "boolean"})
+    add_configs("extras",     {description = "Build extras.", default = true, type = "boolean"})
+    add_configs("use_thread", {description = "Enable threading support.", default = true, type = "boolean"})
```

Comment:
> Update the default values of the options to be consistent with cmake


Please do not modify them. xmake-repo packages are not guaranteed to be consistent with cmake, which has its own package specification and conventions.

In xmake, it is important to ensure minimal compilation and installation, unless necessary, to avoid introducing too many dependencies. Users can enable them themselves.

Also, by modifying them now, you may break some existing users' projects.

Reply:
OK, I will revert these

Reply:
Done